airbyte-ci/connectors/live-tests/src/live_tests/commons/connector_runner.py

```diff
@@ -150,15 +151,15 @@ async def _run(
                     "sh",
                     "-c",
                     " ".join(airbyte_command)
-                    + f"| {self.IN_CONTAINER_OBFUSCATOR_PATH} > {self.IN_CONTAINER_OUTPUT_PATH} 2>&1 | tee -a {self.IN_CONTAINER_OUTPUT_PATH}",
+                    + f"| {self.IN_CONTAINER_OBFUSCATOR_PATH} 2>{self.IN_CONTAINER_STDERR_PATH} 2>&1 | tee -a {self.IN_CONTAINER_STDOUT_PATH}",
                 ],
                 skip_entrypoint=True,
             )
             executed_container = await container.sync()
             # We exporting to disk as we can't read .stdout() or await file.contents() as it might blow up the memory
-            stdout_exported = await executed_container.file(self.IN_CONTAINER_OUTPUT_PATH).export(str(self.stdout_file_path))
+            stdout_exported = await executed_container.file(self.IN_CONTAINER_STDOUT_PATH).export(str(self.stdout_file_path))
             if not stdout_exported:
-                raise errors.ExportError(f"Failed to export {self.IN_CONTAINER_OUTPUT_PATH}")
+                raise errors.ExportError(f"Failed to export {self.IN_CONTAINER_STDOUT_PATH}")
```

Comment:
Shall we also export the `IN_CONTAINER_STDERR_PATH`, might be helpful for debugging.
I'm not 💯 sure but I think `stderr` might also contain airbyte messages, so this change might mean our tests will not consume them.